x-pack/plugins/endpoint/public/applications/endpoint/view/alerts/index.tsx

```diff
@@ -156,7 +156,7 @@ export const AlertIndex = memo(() => {
       } else if (columnId === 'host_name') {
         return row.host.hostname;
       } else if (columnId === 'timestamp') {
-        const date = datesForRows.get(row)!;
+        const date = datesForRows.get(row as AlertData)!;
```

Comment:
@dplumlee This is because `row` is an `ImmutableObject` ... how do you want me to handle this? I'm happy to re-introduce `Immutable*`, but want to make sure it's in line with what you are doing/expecting.